docs/ref/integration/reverse-proxy.md

```diff
@@ -139,3 +139,82 @@ The following minimal Apache config will proxy traffic to the headscale instance
 	SSLCertificateKeyFile <PATH_CERT_KEY>
 </VirtualHost>
 ```
+
+## nginx proxy manager
+
```

Comment:
- A short introduction (similar to Nginx section) is missing
- Missing code markers: ``` - probably also include a filename if applicable